src/app/globals.js

```diff
@@ -1,6 +1,6 @@
 /** @format */
 
-import { createElement } from '@wordpress/element';
+import { createElement } from '@gutenberg/element';
```

Comment:
I updated it to follow the general pattern where all references in `src` folder point to `@gutenberg/*` instead of using a local copy of `@wordpress/*`. We can update it back to `@wordpress/*` as soon as we have all packages published to `npm`.